### DISCUSSION TITLE
Insecure Inclusion

### DISCUSSION THREAD

**Discussion by GiODi20** [zwave-js/zwave-js-ui#378] [Q&A]
I'm only seeing a Secure Inclusion option. Is there a way to include without security?

From what I understand this is the preferred option for switches and motion sensors.

**Comment by robertsLando** [comment]
> From what I understand this is the preferred option for switches and motion sensors.

Just press on `No` or `Cancel` when the popup about the secure inclusion is shown and a non secure inclusion will start
